templates/shared/combomarkdowneditor.tmpl

```diff
@@ -41,12 +41,23 @@ Template Attributes:
 			</div>
 			<div class="markdown-toolbar-group gt-f1"></div>
 			<div class="markdown-toolbar-group">
-				<span class="markdown-toolbar-button markdown-switch-easymde">{{svg "octicon-arrow-switch"}}</span>
+				<button class="markdown-toolbar-button markdown-switch-monospace" role="switch" data-enable-text="{{.locale.Tr "editor.enable_monospace_font"}}" data-disable-text="{{.locale.Tr "editor.disable_monospace_font"}}">{{svg "octicon-typography"}}</button>
+				<button class="markdown-toolbar-button markdown-switch-easymde">{{svg "octicon-arrow-switch"}}</button>
 			</div>
 		</markdown-toolbar>
 		<text-expander keys=": @">
 			<textarea class="markdown-text-editor js-quick-submit"{{if .TextareaName}} name="{{.TextareaName}}"{{end}}{{if .TextareaPlaceholder}} placeholder="{{.TextareaPlaceholder}}"{{end}}{{if .TextareaAriaLabel}} aria-label="{{.TextareaAriaLabel}}"{{end}}>{{.TextareaContent}}</textarea>
 		</text-expander>
+		<script>
+			const monospaceEnabled = localStorage?.getItem('editor-monospace') === 'true';
+			const btn = document.querySelector('.markdown-switch-monospace');
```

Comment:
I guess it still needs longEnoughName to prevent from global name conflicting.

Reply:
Can't be `module` as it would load too late.

Reply:
Variable is completely removed now.